src/main/java/seedu/edulog/ui/StudentCard.java

```diff
@@ -40,6 +42,13 @@ public class StudentCard extends UiPart<Region> {
     private Label email;
     @FXML
     private FlowPane tags;
+    @FXML
+    private ImageView paidStatusIcon;
+
+    private final Image paidIcon = new Image(getClass().getResource("/images/paid.png").toExternalForm());
```

Comment:
Do these images need the @FXML tag as well?

Reply:
Nope. @FXML tags are used for javaFX components so that the controller can check if the component exists on the actual stage. Image is just an internal Object. The component hosting the image is ImageView which does have the @FXML tag